app/views/hiring_staff/vacancies/review.html.haml

```diff
@@ -1,5 +1,4 @@
-- content_for :page_title_prefix, "Review the job details — Publish a job for #{@school.name}"
-
+- content_for :page_title_prefix, "#{@vacancy.errors.present? ? 'Error: ' : ''}Review the job details — Publish a job for #{@school.name}"
 =render partial: 'school_vacancy_breadcrumb'
```

Comment:
Can the title and error be extracted to the locale.
I wonder if it the logic for this can also be moved in the VacancyPresenter as `review_page_title` so we can clean up the view.